### PR TITLE
[DOCS-4942] Add Cloud SIEM Security Signal doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

Adds the new Cloud SIEM Security Signal doc.

[DOCS-4942](https://datadoghq.atlassian.net/browse/DOCS-4942)

### Merge instructions

- [ ] Please merge after reviewing

[DOCS-4942]: https://datadoghq.atlassian.net/browse/DOCS-4942?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/security/cloud_siem/security_signal.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    text: "Learn about the conditional logic of detection rules"
+  - link: "https://www.datadoghq.com/blog/monitor-1password-datadog-cloud-siem/"
+    tag: "Blog"
+    text: "Monitor 1Password with Datadog Cloud SIEM"
+---
+
+## Overview
+
+A Cloud SIEM security signal is created when Datadog detects a threat based on a security rule. View, search, filter, and correlate security signals in the Signal Explorer without needing to learn a dedicated query language. You can either monitor signals from Datadog Security, or configure [Notification Rules][1] to send signals to third-party tools. Signals can be assigned to yourself or another user in the Datadog platform.
+
```
Comment (maycmlee) [inline content/en/security/cloud_siem/security_signal.md:17]: Could add an image of Cloud SIEM security signal page here.

File: content/en/security/cloud_siem/security_signals.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### Workflow automation
+
+You can trigger a Workflow automatically for any Security Signal. You can also manually trigger a Workflow from a Cloud SIEM Security Signal. See [Trigger a Workflow from a Security Signal][7] for more information.
+
+### Threat intelligence
+
+Datadog Cloud SIEM's threat intelligence feeds are curated by threat intelligence partners. These feeds are constantly updated to include data about known suspicious activity (for example, indicators of compromise or IOCs), so you can quickly identify which potential threats to address.
+
+{{< img src="security/security_monitoring/explorer/threat_intel.png" alt="Threat Intelligence in the Security Signals Explorer" style="width:85%;" >}}
```
Comment (maycmlee) [inline content/en/security/cloud_siem/security_signals.md:70]: Image needs to be updated.

File: content/en/security/cloud_siem/investigate_a_security_signal.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    tag: "Documentation"
+    text: "Learn about the conditional logic of detection rules"
+  - link: "https://www.datadoghq.com/blog/monitor-1password-datadog-cloud-siem/"
+    tag: "Blog"
+    text: "Monitor 1Password with Datadog Cloud SIEM"
+---
+
+## Overview
+
+A Cloud SIEM security signal is created when Datadog detects a threat while analyzing logs against detection rules. View, search, filter, and correlate security signals in the Signal Explorer without needing to learn a dedicated query language. You can also correlate, summarize, and assign security signals to yourself or another user in the Datadog platform. Additionally, configure [Notification Rules][1] to send signals to specific individuals or teams to keep them informed of issues.
```
Comment (maycmlee) [inline content/en/security/cloud_siem/investigate_a_security_signal.md:16]:
```suggestion
A Cloud SIEM security signal is created when Datadog detects a threat while analyzing logs against detection rules. View, search, filter, and correlate security signals in the Signal Explorer without needing to learn a dedicated query language. You can also assign security signals to yourself or another user in the Datadog platform. In addition to the Signal Explorer, you can configure [Notification Rules][1] to send signals to specific individuals or teams to keep them informed of issues.
```

File: content/en/security/cloud_siem/investigate_a_security_signal.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  - link: "https://www.datadoghq.com/blog/monitor-1password-datadog-cloud-siem/"
+    tag: "Blog"
+    text: "Monitor 1Password with Datadog Cloud SIEM"
+---
+
+## Overview
+
+A Cloud SIEM security signal is created when Datadog detects a threat while analyzing logs against detection rules. View, search, filter, and correlate security signals in the Signal Explorer without needing to learn a dedicated query language. You can also assign security signals to yourself or another user in the Datadog platform. In addition to the Signal Explorer, you can configure [Notification Rules][1] to send signals to specific individuals or teams to keep them informed of issues.
+
+You must have the `Security Signals Write` permission to modify a security signal, such as change the state and view signal action history in [Audit Trail][2]. See [Role Based Access Control][3] for more information about Datadog's default roles and granular role-based access control permissions available for Datadog Security in the Cloud Security.
```
Comment (maycmlee) [inline content/en/security/cloud_siem/investigate_a_security_signal.md:18]: Leaving this here for now. The plan is to pull this out and have a general RBAC doc for all security products.

**Review by aliciascott — APPROVED** [review]
LGTM!

**Pull request merged**